### ISSUE TITLE
No embed support?

### ISSUE THREAD

**Issue by Cold-Dev** [tsickert/discord-webhook#1]
Hey there.
Great GitHub action, however it appears that it doesn't support embeds in any form due to the fact the "content" field is statically set. Is there any rectification that can be made for this? Some sort of added "embed" field? I'm not quite sure how it'd work as I don't work with GitHub actions often, however it'd be a great addition.

**Comment by tsickert** [comment]
Hey @Cold-Dev, thanks for the question!

TLDR: I've added [v2.0.0](https://github.com/tsickert/discord-webhook/releases/tag/v2.0.0) that supports adding a JSON file to the repository that defines a webhook object. Information on [how to use it](https://github.com/marketplace/actions/discord-webhook-action#advanced-use-cases) was also added. 

For a little more info on why I chose to do a JSON file: the embeds object in the body is an array of objects. 

The obvious choice for adding embeds would have been t take a JSON string as the input. It turns out that passing a JSON string via the shell is not trivial, and any work-arounds just didn't end up feeling good.

Providing a file in the repository creates support for other webhook body fields and allows users to manipulate them before the action is called, which seems like a win-win. The only catch is that the repository needs to be checked out before the webhook action is called.

Hope this solves your problem and thanks again for stopping by!
